addons/hr_work_entry_contract/models/hr_work_entry.py

```diff
@@ -71,6 +100,12 @@ def _onchange_contract_id(self):
         if res.get('contract_id'):
             self.contract_id = res.get('contract_id')
 
+    def _compute_is_missing_work_entry_hours(self):
+        self.is_missing_hours = False
+        work_entries = self._get_missing_work_entry()
+        for work_entry in list(work_entries.keys()):
+            work_entry.is_missing_hours = True
```

Comment:
Same here, you could have this instead:
```suggestion
        work_entries = self._get_missing_work_entry()
        work_entries.is_missing_hours = True
```